.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
**question:** Do you think daily is a healthy interval value? 

I usually choose weekly and am diligent about reviewing once a week Dependabot PRs in other projects.
Maybe for actions and docker not that important, but I'm used to also do it for NPM.

Reply:
Good point, I'll change it to weekly